components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -101,7 +109,10 @@ internal class MediaSessionServiceDelegate(
 
     private fun processMediaSessionState(state: SessionState?) {
         if (state == null) {
-            updateNotification(state)
+            CoroutineScope(notificationDispatcher).launch() {
+                updateNotification(state)
+            }
```

Comment:
If we create a scope every time we want to launch something then it's basically the same as GlobalScope. Here we should create one scope in `onCreate()` and close it in `onDestroy()`. This way everything running asynchronously stops together with the service.

Reply:
Understood.  I'll fix.  Thanks,